jetstream/engine/engine_api.py

```diff
@@ -200,7 +253,15 @@ def get_prefix_destination_sharding(self) -> Any:
   def get_tokenizer(
       self,
   ) -> tokenizer_pb2.TokenizerParameters:
-    """Returns the info to construct a sentencepiece tokenizer in py/c++."""
+    """Returns the info to construct a tokenizer in py/c++."""
+
+  # pylint: disable=unused-argument
+  def build_tokenizer(
+      self,
+      metadata: tokenizer_pb2.TokenizerParameters,
+  ) -> Tokenizer:
+    """Builds a new tokenizer object and returns it."""
+    return None
```

Comment:
maybe add a default impl here to make it not BC breaking

Reply:
Did this to avoid circular import issue. In orchestrator.py, if tokenizer is None, we build the default SentencePieceTokenizer